lab02/sections/discussion.tex

```diff
@@ -1 +1,14 @@
 \section{Discussion}\label{sec:discussion}
+A phase-lag circuit was built of an identical topology to that in Figure 1.
+The frequency of a 1V peak-to-peak input signal was varied over ~12kHz in 1kHz increments. 
```

Comment:
Welcome to Latex weirdness! 

Things that you'd think give you ~ but don't:
- `~` is a special character related to character spacing
- `\~` puts a tilde over a character.
- `\textasciitilde` will _almost_ get you what you want, but it's, uh, too high

Things that actually work:
- `$\sim`, but it's pretty big
- `$\approx` is what I use!

I'd write this as `$\approx \SI{12}{\kilo\hertz}$ in 1 kHz increments.` What's the \SI for? It allows units to appear non-italic and properly spaced in math mode (`$2 hz$` vs `$2 \si{hertz}`). You can use it as `\SI{number}{\units}` or `\si{units}`. 
Honestly, it's overkill a lot of the time, like when you're just referencing values in a discussion but it's really useful when you start mixing equations and values.